### ISSUE TITLE
Status: expose external address

### ISSUE THREAD

**Issue by odacremolbap** [triggermesh/knative-sources#88]
External address URL is not being filled:

```sh
 kubectl get slacksources.sources.triggermesh.io -n tzununbekov        
NAME        READY   REASON   URL   SINK                                                 AGE
slack-bot   True                   http://event-display.tzununbekov.svc.cluster.local   101m
```

As a workaround the URL can be retrieved from the underlying ksvc (when a ksvc is being used)
```sh
 kubectl get ksvc -n tzununbekov slacksource-slack-bot 
NAME                    URL                                                     LATESTCREATED                 LATESTREADY                   READY   REASON
slacksource-slack-bot   https://slacksource-slack-bot.tzununbekov.dev.munu.io   slacksource-slack-bot-j4n8n   slacksource-slack-bot-j4n8n   True    
```

**Comment by antoineco** [comment]
@odacremolbap closing because fixed by #58. The URL is populated properly now:

```console
$ kubectl -n antoineco get slacksources.sources.triggermesh.iot
NAME   READY   REASON   URL                                                   SINK                                                                         AGE
demo   True             https://slacksource-demo.antoineco.k.triggermesh.io   http://broker-ingress.knative-eventing.svc.cluster.local/antoineco/default   1m
```

**Comment by antoineco** [comment]
Reopening, I managed to reproduce:

```console
$ kubectl -n antoineco get zendesksource,ksvc
NAME                                                 READY   REASON   URL   SINK                                               AGE
zendesksource.sources.triggermesh.io/zendesksource   True                   http://event-display.antoineco.svc.cluster.local   51m

NAME                                                      URL                                                              LATESTCREATED                       LATESTREADY                         READY   REASON
service.serving.knative.dev/zendesksource-zendesksource   https://zendesksource-zendesksource.antoineco.k.triggermesh.io   zendesksource-zendesksource-00003   zendesksource-zendesksource-00003   True
```
